spec/sections_spec.rb

```diff
@@ -22,6 +28,24 @@ class Page < SitePrism::Page
     it { is_expected.to respond_to(:has_single_section?) }
   end
 
+  describe '.section with class and block' do
+    before do
+      expect(subject).to receive(:find_first).and_return(:element)
```

Comment:
should this be allow?

Reply:
allow will not make assertion method is called